### DISCUSSION TITLE
Total cost in energy panel is at least 10x what it should be

### DISCUSSION THREAD

**Discussion by inertia666** [custom-components/nordpool#308] [General]
I applied the latest HACS update today (0.0,11) in HA 2023.01.27 and now my energy panel is showing the total price in SEK as much larger than it was before the upgrade.

This is second time this has happened to me and I couldn't repair it but had to do a fresh HA install. Nothing has changed in my config for the units and the nordpool sensor seems to be showing the correct prices.

My energy usage for today (so far) is 12.42kWh and the total accumulated price for today is SEK 667.38! I haven't tallied up the hourly rates but even if it was 67.38 is feels a bit extreme for 12kwh.

**Comment by Rantaki** [comment]
I had similar issue before I noticed that the energy panel takes Euros instead of cents. Maybe you have similar problem (but with SEK)?

**Reply by inertia666** [reply]
Actually, i think the problem was related to Docker. For some reason, it was running multiple versions of the same container and I think it was recording the same data several times to the database.
